eva-server/src/test/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2IntegrationTest.java

```diff
@@ -125,14 +126,31 @@ public void annotationEndPointTestExisting() throws URISyntaxException {
     }
 
     @Test
-    public void annotationEndPointTestNonExisting() throws URISyntaxException {
+    public void annotationEndPointTestNonExistingWithNonExistingVariant() throws URISyntaxException {
```

Comment:
I don't understand the name of this method. If you are trying to test one case where only the variant does not exist, and another where neither the variant nor annotation exist, you can create two methods and name each one more specifically.

Reply:
Fixed :)